packages/monorepo-tools/where.js

```diff
@@ -52,7 +53,7 @@ async function lernaExec(packages) {
     return;
   }
 
-  const lernaBin = path.resolve(process.cwd(), 'node_modules', '.bin', 'lerna');
+  const lernaBin = path.resolve(MONOREPO_ROOT, 'node_modules', '.bin', 'lerna');
```

Comment:
I need to test the useLernaExec=true path still..

Reply:
Why is this still needed?

Reply:
we get packages as argument already:

``` js
for packageInfo of packages {
   await util.promisify(execFile)(execCommandArgs[0], ...execCommandArgs.slice(1), {cwd: packageInfo.location})
}
```

Reply:
I'm not sure what you're referring to. I just ported over the script and it has a path where it does what you copy/pasted and one where it uses lerna.